BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java

```diff
@@ -608,10 +600,10 @@ public void removeItem(BaseBrick item) {
 
             if (brickRecyclerAdapter != null) {
                 brickRecyclerAdapter.safeNotifyItemRemoved(index);
-                LinkedList<BaseBrick> items = getRecyclerViewItems();
+                LinkedList<BaseBrick> recyclerViewItems = getRecyclerViewItems();
```

Comment:
can this be List<>?

Reply:
Yeah, I was gonna do more of the LinkedList -> List API changes all at once